CRM/Contribute/BAO/Query.php

```diff
@@ -386,6 +386,13 @@ public static function whereClauseSingle(&$values, &$query) {
         $query->_tables['civicrm_contribution_recur'] = $query->_whereTables['civicrm_contribution_recur'] = 1;
         return;
 
+      case 'contribution_recur_contribution_status_id':
+        $query->_where[$grouping][] = CRM_Contact_BAO_Query::buildClause("civicrm_contribution_recur.contribution_status_id", $op, $value, 'String');
+        list($op, $value) = CRM_Contact_BAO_Query::buildQillForFieldValue('CRM_Contribute_DAO_ContributionRecur', 'contribution_status_id', $value, $op, $pseudoExtraParam);
+        $query->_qill[$grouping][] = ts('%1 %2 %3', array(1 => 'Recurring Contribution Status', 2 => $op, 3 => $value));
```

Comment:
Please add single quotation around status value to highlight the statuses, because it qill odd for ```In progress``` status : 
![screen shot 2017-10-13 at 11 33 44 am](https://user-images.githubusercontent.com/3735621/31532448-03fa0bac-b00b-11e7-8b25-36ecdd52a3dd.png)

Reply:
Sure thing. 

I was wondering if I should quote the status values when I developed this field, but when I checked the output for normal contribution status (not recurring), there was no quote. So I just followed core behaviour.

![contribution status output](https://user-images.githubusercontent.com/374068/31553390-91f4c780-aff7-11e7-8998-ff964a69261f.png)